projects/chubaofs/cubefs-graduation-dd.md

```diff
@@ -0,0 +1,504 @@
+  > In particular, CubeFS enables the separation of storage/compute architecture for databases and AI/ML applications.
+  > 
+  > Some key features of CubeFS include:
+  > 
+  > - Multiple access protocols such as POSIX, HDFS, S3, and its own REST API
+  > - Highly scalable metadata service with strong consistency  
+  > - Performance optimization of large/small files and sequential/random writes
+  > - Multi-tenancy support with better resource utilization and tenant isolation
+  > - Hybrid cloud I/O acceleration through multi-level caching
+  > - Flexible storage policies, high-performance replication or low-cost erasure coding
+
+- [x] **Document what the project does, and why it does it - including viable cloud native use cases.**
+
+  CubeFS Introduction: [link](https://cubefs.io/docs/master/overview/introduction.html)
+
+  According to [CubeFS doc](<https://cubefs.io/docs/master/overview/introduction.html#application-scenarios>), typical use cases are:
+  - Big Data Analytics
+  - Deep Learning/Machine Learning
+  - Container Shared Storage
+  - Database & Middleware
+  - Online Services
+  - Traditional NAS to Cloud
+
+- [x] **Document and maintain a public roadmap or other forward looking planning document or tracking mechanism.**
+
+  CubeFS has a public roadmap doc at [ROADMAP.md](https://github.com/cubefs/cubefs/blob/206d5ddadf1f99abde6401b7aa18b57fc46e6bed/ROADMAP.md)
+
+- [x] **Roadmap change process is documented.**
+
+  CubeFS documentes its roadmap rules and changing process in [GOVERNANCE.md#roadmap](https://github.com/cubefs/cubefs/blob/206d5ddadf1f99abde6401b7aa18b57fc46e6bed/GOVERNANCE.md#roadmap)
+
+- [x] **Document overview of project architecture and software design that demonstrates viable cloud native use cases, as part of the project's documentation.**
+
+  - CubeFS architecture overview: <https://cubefs.io/docs/master/overview/architecture.html#architecture>
+  - Design details are documented under the "Design" section, an example is: <https://cubefs.io/docs/master/design/master.html>
+
+- [x] **Document the project's release process and guidelines publicly in a RELEASES.md or equivalent file that defines:**
+
+  - [x] **Release expectations (scheduled or based on feature implementation)**
+
+    CubeFS documents their release frequency as needed (beta and official releases), which can be regarded as based on feature implementation.
+  - [x] **Tagging as stable, unstable, and security related releases**
+
+    CubeFS uses beta to mark their unstable releases. Ref: [RELEASE.md#types-of-releases](https://github.com/cubefs/cubefs/blob/206d5ddadf1f99abde6401b7aa18b57fc46e6bed/RELEASE.md#types-of-releases).
+    Security release process is documented at: [security-release-process.md](https://github.com/cubefs/cubefs/blob/206d5ddadf1f99abde6401b7aa18b57fc46e6bed/security/security-release-process.md)
```

Comment:
@kevin-wangzefeng from the [fix development process documentation](https://github.com/cubefs/cubefs/blob/206d5ddadf1f99abde6401b7aa18b57fc46e6bed/security/security-release-process.md#fix-development-process) it looks like [tagging](https://github.com/cubefs/cubefs/tags) isn't used for bug fixes or security fixes. Recommend the project use tagging so it's easier for end users to identify which releases are most important to upgrade to - especially for enterprise end users.

Reply:
changes look great